bundler/ruby_version_resolver.go

```diff
@@ -0,0 +1,45 @@
+package bundler
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+// RubyVersionResolver identifies and compares versions of Ruby used in the
+// build environment.
+type RubyVersionResolver struct {
+}
+
+// NewRubyVersionResolver initializes an instance of RubyVersionResolver.
+func NewRubyVersionResolver() RubyVersionResolver {
+	return RubyVersionResolver{}
+}
+
+// Lookup returns the version of Ruby installed in the build environment.
+func (r RubyVersionResolver) Lookup(workingDir string) (string, error) {
+	getRubyVersionCmd := strings.Join([]string{
+		"rvm",
```

Comment:
the RVM Buildpack uses a different logic to determine the Ruby version. Do you see a potential risk coming from the fact that RVM and RVM Bundler use different Ruby version determination logics?